src/dialogs/PaymentErrorDialog.vue

```diff
@@ -28,21 +28,19 @@
         <!-- display errors -->
         <div class="font-15 mb-4" v-if="numErrors > 0">
           <p>We were unable to process your payment due to the following errors:</p>
-          <ul>
-            <li v-for="(error, index) in errors" :key="index">
-              {{error.message}}
-            </li>
-          </ul>
+          <div v-for="(error, index) in errors" :key="index" class="d-flex">
+            <span class="flex-shrink-0"><v-icon>mdi-circle-medium</v-icon></span>
+            <span class="flex-grow-1" v-html="error.message" />
+          </div>
         </div>
 
         <!-- display warnings-->
         <div class="font-15 mb-4" v-if="numWarnings > 0">
           <p>Please note the following warnings:</p>
-          <ul>
-            <li v-for="(warning, index) in warnings" :key="index">
-              {{warning.message}}
-            </li>
-          </ul>
+          <div v-for="(warning, index) in warnings" :key="index" class="d-flex">
+            <span class="flex-shrink-0"><v-icon>mdi-circle-medium</v-icon></span>
+            <span class="flex-grow-1" v-html="warning.message" />
+          </div>
```

Comment:
Removed class and used default css.

Reply:
Thanks for the comments. You don't have to comment all code changes; only the ones that need an explanation :)